src/parser_urdf.cc

```diff
@@ -3484,7 +3484,7 @@ void ReduceSDFExtensionPluginFrameReplace(
         // create new offset xml blocks
         xyzKey = new TiXmlElement("xyzOffset");
         rpyKey = new TiXmlElement("rpyOffset");
-        correctedOffsetKey = new TiXmlElement("ignition::corrected_offsets");
+        correctedOffsetKey = new TiXmlElement("gz::corrected_offsets");
```

Comment:
these changes to the element name should be reverted

Reply:
[fb6d5f8](https://github.com/gazebosim/sdformat/pull/1118/commits/fb6d5f826605b46774720042e9fa6110baee5090)